apps/frontend/core-dapp/src/stores/DepositTradeHelperStore.ts

```diff
@@ -241,69 +216,41 @@ export class DepositTradeHelperStore extends ContractStore<RootStore, SupportedC
     }
   }
 
-  async withdrawAndUnwrap(
-    recipient: string,
-    amountIn: BigNumber,
+  createWithdrawAndUnwrapTx({
+    amountIn,
+    amountOut,
+    permit,
+    recipient,
+  }: {
+    amountIn: BigNumber
     amountOut: BigNumber
```

Comment:
probably more readable imo if we simply do named parameters here since the object is neither reused inside/outside the function, but feel free to disregard. if you do this however make sure to also update other functions for consistency

```suggestion
  createWithdrawAndUnwrapTx(
    amountIn: BigNumber,
    amountOut: BigNumber,
    permit: SignedPermit,
    recipient: string
  ): UnsignedTxOutput {
```

Reply:
I think in this case what we lose in readability here (which isn't a lot IMO) we gain in readability on the usage site (which is more significant)

The current call code:

```ts
return this.root.depositTradeHelperStore.createWithdrawAndUnwrapTx({
  amountIn: this.withdrawalAmount.inWstEthBN,
  amountOut: this.withdrawalAmount.inEthBN,
  permit,
  recipient: address,
})
```

Would turn into:

```ts
return this.root.depositTradeHelperStore.createWithdrawAndUnwrapTx({
  this.withdrawalAmount.inWstEthBN,
  this.withdrawalAmount.inEthBN,
  permit,
  address,
})
```

Which can be more confusing (what's the amount in? what's the amount out? whose `address` is that?)